ports/indicators/portfile.cmake

```
@@ -3,8 +3,8 @@
 vcpkg_from_github(
     OUT_SOURCE_PATH SOURCE_PATH
     REPO p-ranav/indicators
-    REF ec1973607d5b360b3d1f9b07af851ce236ebd0ae
-    SHA512 9b9678129525f03ece91217eb71098b8e14f76f178f9abfd24aa0679bcf7ae27dd600bdba185987e9908dfd8fb5e837dc903a8be011fb08d8707216a3084522a
+    REF 755900d

```

```suggestion
    REF a776a52c983599f81dc79eaee670acd9757af730 # v1.9
```
The REF should be the latest commit id that this release has. Why do you use another one?

---

[indicators#64](https://github.com/p-ranav/indicators/issues/67) was fixed by this commit, and so the latest one for the release has the issue and will not build on Windows. Is it better to use a patch in place of a later commit?

---

In this case, I think you can update the port to the latest commit in master branch instead of this release 1.9.